src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -293,6 +282,52 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
             return matchingVersion?.ToString();
         }
 
+        private NuGetVersion ResolvePlatformReleaseChannelVersion(VersionRange versionRange, IEnumerable<string> versions)
+        {
+            var bundleVersions = versions.Select(p =>
+            {
+                var dirName = Path.GetFileName(p);
+                NuGetVersion.TryParse(dirName, out NuGetVersion version);
+                if (version != null)
+                {
+                    version = versionRange.Satisfies(version) ? version : null;
+                }
+                return version;
+            }).Where(v => v != null).OrderByDescending(version => version.Version).ToList();
+
+            return _platformReleaseChannel.ToUpperInvariant() switch
+            {
+                ScriptConstants.StandardPlatformChannelNameUpper or ScriptConstants.ExtendedPlatformChannelNameUpper => GetStandardOrExtendedBundleVersion(bundleVersions),
+                ScriptConstants.LatestPlatformChannelNameUpper or "" => GetLatestBundleVersion(bundleVersions),
+                _ => HandleUnknownPlatformReleaseChannelName(bundleVersions)
+            };
+        }
+
+        private NuGetVersion GetStandardOrExtendedBundleVersion(IList<NuGetVersion> orderedByDescBundlesList)
+        {
+            if (orderedByDescBundlesList.Count > 1)
+            {
+                return orderedByDescBundlesList[1]; // if n is latest, should get version n-1 as this is how the platform release channel defines these channel names (resolve to version prior to latest)
```

Comment:
This comment and the code doesn't line up. Why does it mention `n-1`, but you get `1`?

Reply:
I'm trying to best translate the logic from here: https://github.com/Azure/azure-functions-host/issues/10778

We want the "next latest" or "version prior to latest". I can change the comment to: 
`For these release channel types, we should get the version prior to latest`? 

I'm thinking now that it's more confusing to go into the full logic and just keep it simple?

Reply:
Okay I understand that issue. It is written assuming latest would be `orderedByDescBundlesList[orderedByDescBundlesList.Count - 1]`, but we are ordering them in the opposite direction. We should update all comments in this PR to be based on how the code orders them (and not the ordering the issue assumes).